src/UI/Pages/CompileReports/CompileReportsTableRow.tsx

```diff
@@ -37,7 +36,7 @@ export const CompileReportsTableRow: React.FC<Props> = ({ row }) => {
         <Td dataLabel={words("compileReports.columns.compileTime")}>
           {row.compileTime}
         </Td>
-        <Td dataLabel={words("compileReports.columns.actions")}>
+        <Td width={10}>
```

Comment:
This is not really positioning it at the end of the table. Just making the column narrow.
`modifier="fitContent"` will probably also work here.